cves/CVE-2018-13304.yml

```diff
@@ -90,24 +96,11 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: bd27a9364ca274ca97f1df6d984e88a0700fb235
-  note: ''
-- commit: ''
-  note: ''
+  note:
+
 vccs:
-- commit: 2aa9047486dbff12d9e040f917e5f799ed2fd78b
-  note: Identified by archeogit,
 - commit: f9d3841ae6147eaa51c57c574cd81e9ce9566e3a
   note: Identified by archeogit,
```

Comment:
Might be worth noting that this was introduced when support for this profile was added just to give context without needing to view the commit itself.